tests/acceptance/features/apiSpaces/shareSpacesViaLink.feature

```diff
@@ -0,0 +1,87 @@
+@api @skipOnOcV10
+Feature: Share spaces via link
+    As the manager of a space
+    I want to be able to share a space via public link
+
+    Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+    See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+    Background:
+        Given these users have been created with default attributes and without skeleton files:
+            | username |
+            | Alice    |
+            | Brian    |
+        And the administrator has given "Alice" the role "Space Admin" using the settings api
+        And user "Alice" has created a space "share space" with the default quota using the GraphApi
+        And user "Alice" has uploaded a file inside space "share space" with content "some content" to "test.txt"
+
+
+    Scenario Outline: A manager can share a space to public via link with different permissions
+        When user "Alice" creates public link share of the space "share space" with settings:
+            | shareType   | 3             |
```

Comment:
This line is not really needed. The step text indicates "public link share" and the step implementation code defaults to "3".